fstat/lib.py

```diff
@@ -25,6 +36,28 @@ def parse_end_date(end_date=None):
     return end_date
 
 
+def get_teams():
```

Comment:
Isn't this really a `get_user_orgs`? Would it be worth just making this call inside the `organization_access_required` function? Or are we using this elsewhere?